odoo/modules/loading.py

```diff
@@ -415,6 +415,8 @@ def load_modules(registry, force_demo=False, status=None, update_module=False):
         # STEP 1: LOAD BASE (must be done before module dependencies can be computed for later steps)
         graph = odoo.modules.graph.Graph()
         graph.add_module(cr, 'base', force)
+        for pyfile in tools.config.options.get('upgrade_scripts', '').split(','):
+            odoo.modules.migration.exec_script(cr, graph['base'].installed_version, pyfile, 'base', 'pre')
```

Comment:
- Move it after the `if graph` check.
- Should be in a `if update_module` block.